packages/client/src/app/jobs/executeJobModal.tsx

```diff
@@ -41,7 +44,16 @@ export const ExecuteJobModal = ({ closeModal, devices, jobId }: ExecuteJobModalP
         <Text h3>Execute {jobId}</Text>
       </Modal.Header>
       <Modal.Body>
-        {devices && (
+        <Input
```

Comment:
You should use the search component from https://github.com/UnownHash/Rotom/blob/main/packages/client/src/app/status/search.tsx

Example implementation: https://github.com/UnownHash/Rotom/blob/55ea8fc85e315269136992bacfdce60e6a2f2d18/packages/client/src/app/status/devicesTable.tsx#L78